index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <h1 class="logo">John Doe</h1>
+       <div class="harmburger flex">
+        <span class="line"></span>
+        <span class="line"></span>
+        <span class="line"></span>
+       </div>

```

Comment:
- [ ] The hamburger button should have a nav tag, please add this it's a requirement